packages/legend-application-studio/src/stores/editor-state/element-editor-state/ClassEditorState.ts

```diff
@@ -39,13 +39,16 @@ import {
 export class ClassEditorState extends UMLEditorState {
   classState: ClassState;
   selectedProperty: AbstractProperty | undefined;
+  isBeingDragged = false;
```

Comment:
confused on this property. where do you use this?
Its on the classeditorState which suggests the whole class is being dragged.